recipes-support/boost/boost_%.bbappend

```diff
@@ -1 +1,5 @@
 PACKAGECONFIG_append = " python"
```

Comment:
This patch is only necessary for version 1.58.I think that if you substitute % for the proper version in the filename it will apply just for that version. Could you try that please?